seeders/courses.seeder.js

```diff
@@ -11,14 +11,44 @@ const data = async () => {
   const so6 = await StudentOutcome.findOne({ studentOutcomeNumber: 6 });
   return [
     {
-      title: "Programming1",
-      courseNumber: 1500,
-      studentOutcome: [so1, so2, so3],
+      title: "Programming I",
+      courseNumber: 1375,
+      studentOutcome: [so1, so2],
     },
     {
-      title: "Capstone",
-      courseNumber: 4321,
-      studentOutcome: [so4, so5, so6],
+      title: "Capstone Project",
+      courseNumber: 4392,
+      studentOutcome: [so5, so6],
+    },
+    {
+      title: "First Year Experience for Computing Majors",
+      courseNumber: 1105,
+      studentOutcome: [so1, so3],
+    },
+    {
+      title: "Algorithms",
+      courseNumber: 2380,
+      studentOutcome: [so1, so2],
+    },
+    {
+      title: "Computer Organization",
+      courseNumber: 2482,
+      studentOutcome: [so1, so4],
+    },
+    {
+      title: "Operating Systems",
+      courseNumber: 3380,
+      studentOutcome: [so3, so4],
+    },
+    {
+      title: "Programming Languages",
+      courseNumber: 3383,
+      studentOutcome: [so4, so6],
+    },
+    {
+      title: "Software Engineering",
```

Comment:
I'm nit-picking here, but since this is going to be the seed data displayed to the client, and the client is intimately familiar with the CS curriculum, we should use the full course names whenever possible. I believe the full course name here is Fundamentals of Software Engineering.

Reply:
I had thought the same thing but when I referenced the Course Catalog the course is listed strictly as 4373 Software Engineering. If you would prefer the name be altered then I am okay with that, but I wanted to point that out before making any changes.

Reply:
Nope, if that's what it says in the catalog let's go with that. Thanks for double checking!